### ISSUE TITLE
Section 508 Standards Course Topic 4 video example 

### ISSUE THREAD

**Issue by AnnMarieDavis** [Section508Coordinators/DHSA-TT-200-A#229]
## Issue Type 
- [x] 508 issue (something is not accessible according to our own test process)

## Description
Audio description example given for SC 1.2.3 and 1.2.5 conforming example 2 for Section 508 Standards for Web course does not meet audio description requirements.  The audio description is provided in text on the screen rather than being narrated in the sound track for users who cannot see the video portion of the multi-media. Issue reported by Carmen Matthews,
Accessibility Specialist, Texas Workforce Commission, 101 E. 15th St., Room 0242, Austin, TX 78778 Phone (512) 936-3715 accessibility@twc.state.tx.us

## Location
[https://training.section508testing.net/mod/book/view.php?id=783&chapterid=1092, Topic 4, Chapter, Subchapter]

## Actual Behavior / Observation
[The audio description appears in closed captioned text, but is not available in the sound track. 

## Expected Behavior / Recommendation
[The multimedia example needs to be fully conforming. Audio descriptions need to be available in the narration.]
